db/migrate/20191129082950_create_actors.rb

```diff
@@ -0,0 +1,12 @@
+class CreateActors < ActiveRecord::Migration[6.0]
+  def change
+    create_table :actors do |t|
+      t.string :name
```

Comment:
It is good idea to add database constraints/validations also, so someone can't enter invalid data directly inside the db, for example `t.string :name, null: false`